src/main/java/com/connexta/ddf/catalog/direct/CatalogMethods.java

```diff
@@ -318,9 +318,10 @@ private Object query(Map<String, Object> params) {
     }
 
     SortBy sortPolicy = SortBy.NATURAL_ORDER;
-    if (params.containsKey("sortPolicy")) {
-      if (params.get("sortPolicy") instanceof Map) {
-        Map<String, Object> rawSortPolicy = (Map) params.get("sortPolicy");
+    if (params.containsKey("sortPolicy") && params.get("sortPolicy") instanceof List) {
```

Comment:
Can we update the DocMethod to reflect these changes? 
Here: https://github.com/connexta/ddf-jsonrpc/pull/18/files#diff-4b8f31c7553886d543ac3f68641f3f4eR113